kythe/go/extractors/config/extractor.go

```diff
@@ -29,21 +29,35 @@ import (
 // kytheConfigFileName The name of the Kythe extraction config
 const kytheExtractionConfigFile = ".kythe-extraction-config"
 
+// Repo is a loose container of input/output parameters for doing extraction on
+// remote repositories.
+type Repo struct {
+	// URI is the remote location of the repo itself.
+	URI string
+	// OutputPath is where to write any relevant output from an extraction
+	// call.
+	OutputPath string
+	// ConfigPath is an optional path to a
```

Comment:
```
// An optional path to a file containing a kythe.proto.ExtractionConfiguration
// encoded as JSON that details how to perform extraction.
```

(exact wording not crucial: Key point is to mention the file format, and that this is a file path rather than the object itself)